aws_jobs/cyhy-data-extract.py

```diff
@@ -390,6 +390,8 @@ def main():
                 "agency.location": True,
                 "agency.name": True,
                 "agency.type": True,
+                "agency.contacts": True,
+                "key": True,
                 "children": True,
                 "enrolled": True,
                 "networks": True,
```

Comment:
Preserve alphabetization:
```suggestion
                "agency.contacts": True,
                "agency.location": True,
                "agency.name": True,
                "agency.type": True,
                "children": True,
                "enrolled": True,
                "networks": True,
                "key": True,
```

Reply:
This is no longer an issue, after https://github.com/cisagov/cyhy-feeds/pull/52/commits/e1f73a9d5398388127b376820c50e66bf0f21a18.